vault/__manifest__.py

```diff
@@ -5,12 +5,12 @@
     "name": "Vault",
     "summary": "Password vault integration in Odoo",
     "license": "AGPL-3",
-    "version": "14.0.1.6.0",
+    "version": "14.0.1.6.1",
```

Comment:
This change will be done by the bot, please restore to the previous version

Reply:
Done